packages/analytics-js-plugins/src/nativeDestinationQueue/utilities.ts

```diff
@@ -13,29 +13,34 @@ import { DEFAULT_QUEUE_OPTIONS } from './constants';
 const getNormalizedQueueOptions = (queueOpts: DestinationsQueueOpts): DestinationsQueueOpts =>
   mergeDeepRight(DEFAULT_QUEUE_OPTIONS, queueOpts);
 
-const isEventDenyListed = (eventName: Nullable<string>, dest: Destination) => {
-  if (!eventName || typeof eventName !== 'string') {
+const isEventDenyListed = (eventType: string, eventName: Nullable<string>, dest: Destination) => {
```

Comment:
`eventType` should be of `RudderEventType` type.

Reply:
Addressed